internal/packer_registry/env/env.go

```diff
@@ -37,3 +31,8 @@ func InPARMode() bool {
 
 	return true
 }
+
+func IsPAREnabled() bool {
+	_, ok := os.LookupEnv(HCPPackerRegistry)
```

Comment:
I think we need to validate here. If the value is `0` or `OFF` then return false, right?

Reply:
You're right. I'll make sure to check if is different from these two possible values.